### ISSUE TITLE
Allow "webidl" css class on pre tags to denote a WebIDL block

### ISSUE THREAD

**Issue by janiceshiu** [w3c/respec#2614]
## Important info
First discussed [here](https://github.com/w3c/respec/pull/2611#discussion_r355242835)

```HTML
<pre class="webidl">
dictionary D{};
</pre> 
```
We'd like the code above ^ to have webIDL syntax highlighting. Right now, it is highlighted as a  generic code block. 

```HTML
<pre class="idl">
dictionary D {};
</pre>
```
^ `pre.webidl` blocks should behave similarly to `pre.idl` blocks, which already behave in the intended way.

### To get started
* https://github.com/w3c/respec/blob/91232584dd2163aa6a9222709e79fe8152a17b3a/src/core/webidl.js#L2 is what respec currently supports.
* https://github.com/w3c/respec/blob/91232584dd2163aa6a9222709e79fe8152a17b3a/src/core/webidl.js#L354-L358 is where respec matches the classes.
* It may also require changes to the style sheet that controls WebIDL syntax highlighting to support the top-level `.webidl` class

**Comment by ridhishjain** [comment]
@janiceshiu @saschanaz I would like to work on this issue

**Comment by ridhishjain** [comment]
I would like to work on this issue.
`export async function run() {`
  `const idls = document.querySelectorAll("pre.idl");`
  `const webidls = document.querySelectorAll("pre.webidl");`
  `...(changes)`
`}`
Can the function present in the `src/core/webidl.js` be made to work similarly for `"pre.webidl"` class like the above code?

**Comment by saschanaz** [comment]
Yup, this is fixed in #2627. 🎉
